independent-publisher-connectors/AzureADApplications/apiProperties.json

```diff
@@ -0,0 +1,61 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "",
+          "scopes": [],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "AzureActiveDirectoryResourceId": "https://graph.microsoft.com"
+          },
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "tenantId": {
+              "value": "common"
+            },
+            "resourceUri": {
+              "value": "https://graph.microsoft.com"
+            }
+          }
+        }
+      },
+      "token:TenantId": {
+        "type": "string",
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "uiDefinition": {
+          "constraints": {
+            "required": "false",
+            "hidden": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0072C6",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "setheader",
+        "title": "Apply consistency level in header",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "ConsistencyLevel",
+          "x-ms-apimTemplateParameter.value": "eventual",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request",
+          "x-ms-apimTemplate-operationName": [
+            "ListApplications"
+          ]
+        }
+      }
+    ],
+    "publisher": "Paul Culmsee"

```

Comment:
@paulculmsee Please add `stackOwner` as well.

Documentation - [Adding metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4-add-metadata)

Reply:
Sorry - corrected and pushed